src/components/ListItem/ListItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './ListItem.css';
+import '../../components/Item/Item';
+import Item from '../../components/Item/Item';
+function ListItem() {
+  return (
+    <div id = "ListCenter">   
+      <ul className="container flex">
```

Comment:
There are some ready made List components. Consider using those instead. No need for change now but it might be easier to do now than change later.

The ready made list components usually takes an array of items and have features like selecting the items.

Some examples can be found from https://reactjsexample.com/tag/list/